src/rp2_common/hardware_watchdog/include/hardware/watchdog.h

```diff
@@ -80,6 +86,17 @@ void watchdog_enable(uint32_t delay_ms, bool pause_on_debug);
  */
 bool watchdog_caused_reboot(void);
 
+/**
+ * \brief Did watchdog_enable cause the last reboot?
+ * \ingroup hardware_watchdog
+ *
+ * This method checks watchdog scratch register 4 for a magic number set by \ref watchdog_enable. This will not be present
+ * if the watchdog reset is initiated by \ref watchdog_reboot or by the RP2040 bootrom (e.g. dragging a UF2 onto the RPI-RP2 drive)
+ * @return true if the watchdog timer or a watchdog force caused the last reboot and the watchdog was initiated by \ref watchdog_enable
+ * @return false there has been no watchdog reboot since run has been
```

Comment:
"since run has been" ??

Reply:
Still not clear what "since run has been" is supposed to mean, and I'm having a hard time interpreting the `@return true` part too. I'll have another look in the morning when I'm less tired.

Reply:
@Wren6991 @liamfraser  "watchdog run has been" was in the comments at release - perhaps a good sentence which described whatever resets the watchdog reboot flag

Reply:
New wording makes more sense to me